src/app/layout/navbar/Navbar.component.tsx

```diff
@@ -27,17 +30,19 @@ const Navbar = () => {
                         />)
                 }
             </ul>
-            <div className="nav_auth_wrapper">
+            <div className={styles.navbar__auth_wrapper}>
                 <DefaultButton
-                    service={() => console.log("elo")}
-                    isFilled={false}
-                    content={"Zaloguj"} />
+                    fillType={"white"}
+                    content={"Zaloguj"}
+                    href={"/login"}
+                    fontSize={"fsmall"} />
```

Comment:
If you use normal string then don't put them in brackets.  

```suggestion
                    fillType="white"
                    content="Zaloguj"
                    href="/login"
                    fontSize="fsmall" />
```